maas-api.yaml

```diff
@@ -1,555 +1,677 @@
-                    realtime:
-                      - alerts: false
-                      - positions: true
-                      - updates: true
-              - id: 27
-                name: LADOT
-                modes:
-                  - mode: pt_pub_bus
-                    numberOfServices: 15828
-              - id: 6216179
-                name: Big Blue Bus
-                modes:
-                  - mode: pt_pub_bus
-                    numberOfServices: 7702
-                    realtime:
-                      - alerts: true
-                      - positions: true
-                      - updates: false
+              type: string
+        - name: onlyRealTime
+          in: query
+          description: Boolean to filter only agencies with real-time support. When true, returned agency list will contain only PTA with Real Time service.
+          required: false
+          schema:
+            type: boolean
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ArrayOfAgencies'
         400:
           description: Bad request, most likely due to issues with the input or non-posible scenarios.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InputError'
+
+  /regions/{id}/alerts:
+    get:
+      tags:
+        - Public Transport
+        - Real-time
+      summary: Real-time alerts
+      parameters:
+        - name: id
+          in: path
+          required: true
           schema:
-            $ref: '#/definitions/InputError'
+            type: string
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  alerts:
+                    $ref: '#/components/schemas/AlertsInfo'
+                required:
+                  - alerts
+        400:
+          description: Bad request due to bad input data
 
-  /info/route.json:
-    post:
+  /agencies/:
+    get:
+      tags:
+        - Public Transport
+      summary: All agencies
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ArrayOfAgencies'
+
+  /agencies/{aid}:
+    get:
+      tags:
+        - Public Transport
+      summary: Public transport agency
+      parameters:
+        - name: aid
+          in: path
+          required: true
+          description: Agency ID
+          schema:
+            type: string
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Agency' 
+
+  /agencies/{aid}/routes:
```

Comment:
can we move this to `/routes` and have a parameter for `aid`?

Reply:
I prefer the current way (also suggested by Mariano), as the hierarchical structure of the URL follows REST.

Reply:
API convention is that `GET /routes/{some id}` should give the route of that ID. Or do you just mean having `GET /routes/?agencyId={aid}`?

Reply:
Played around with it and agree that this looks nicer overall in the specs. Will change it. @MarianoTucat, is that okay with you?

Reply:
yes, sounds good to me

Reply:
oh i don't like the GET & POST for the same URL taking different parameters.  can we do something about that?  (like allow POST only for v1)

same question for `/services`